apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/repo.ex

```diff
@@ -57,6 +57,15 @@ defmodule Astarte.DataUpdaterPlant.Repo do
     end
   end
 
+  def safe_delete_all(queryable, opts \\ []) do
+    try do
+      delete_all(queryable, opts)
```

Comment:
nit: 
if you wrap this in a result, it makes the match easier (you can drop the `when is_integer` check)

```suggestion
     {:ok, delete_all(queryable, opts)}
```